code/game/objects/items/cards_ids.dm

```diff
@@ -122,6 +122,7 @@
 	var/access_txt // mapping aid
 	var/datum/bank_account/registered_account
 	var/obj/machinery/paystand/my_store
+	var/noaccount
```

Comment:
this adds a lot of meaningless var to all types of cards, while this function is too simple.
I think you can just override new `Initialize()` than adding a new var.

Reply:
ah, my bad. I thought it was from `Initialize()` not from `examine()`

Okay, don't use `if(!noaccount)`. use `if(istype(src, /obj/item/card/id/paper))` instead.